### PR TITLE
fix(useVideoStream): error handling & add logs.

### PR DESCRIPTION
**Pull request by hristoterezov**
Add try catch when executing the task in TaskQueue. This should prevent the queue for being blocked if the task throws an exception.

Add more logging around TaskQueue in order to be able to debug reports for blocked queue. 

Add more logging around useVideoStream in order to be able to debug reports of video unmute not executing.

Depends on https://github.com/jitsi/lib-jitsi-meet/pull/1514

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -1020,7 +1020,11 @@ export default {
                     // Rollback the video muted status by using null track
                     return null;
                 })
-                .then(videoTrack => this.useVideoStream(videoTrack));
+                .then(videoTrack => {
+                    logger.debug(`muteVideo: calling useVideoStream for track: ${videoTrack}`);
```
Comment (saghul) [inline conference.js:1024]: Please don't log full tracks, it's incredibly verbose on Mobile. Yes, I know this specific part of the code does not run on mobile, but I'd rather us not have the habit. What are we interested in, about the track?

**Review by saghul — APPROVED** [review]
LGTM with a comment.

File: conference.js (hunk 1 above)
Comment (hristoterezov) [reply conference.js:1024]: We are interested to see what parameters do we use for the function. 

Out of curiosity, why printing the jitsiTrack is verbose on mobile. I see we have defined `toString` method and it prints something like `LocalTrack[3,video]`. Am I missing something?

Comment (paweldomas) [reply conference.js:1024]: yes, `JitsiTrack` is okay to be logged as a string

Comment (saghul) [reply conference.js:1024]: Ah sorry for the noise then! Maybe it's the Jingle IQ that I confused it with. We do have some verbose objects being logged. Anyway, carry on! :-)

**Pull request merged**